src/Integration.Vsix/SonarLintTagger/TextBufferIssueTracker.cs

```diff
@@ -192,7 +193,7 @@ public IEnumerable<ITagSpan<IErrorTag>> GetTags(NormalizedSnapshotSpanCollection
         public void Dispose()
         {
             document.FileActionOccurred -= SafeOnFileActionOccurred;
-            textBuffer.Properties.RemoveProperty(typeof(IIssueTracker));
+            textBuffer.Properties.RemoveProperty(TaggerProvider.SingletonManagerPropertyCollectionKey);
```

Comment:
I've added a test to check that this is set correctly by the `TaggerProvider` and removed by the TBIT.